language/vm/vm_runtime/src/data_cache.rs

```diff
@@ -246,8 +246,12 @@ impl<'txn> TransactionDataCache<'txn> {
                 let deleted = global_ref.is_deleted();
                 if let Some(data) = global_ref.get_data() {
                     if deleted {
+                        // The write set will not grow to this size due to gas limits
```

Comment:
Is there a sensible multiple of a gas limit constant we can refer to in the assumption here?
If not, how is this correct?
> // The write set will not grow to this size due to gas limits

If so, why not refer to it instead of `usize::max_value` ?

Reply:
Adding a comment to explain this.